autorest/utils/errors.go

```diff
@@ -0,0 +1,14 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+)
+
+// OnErrorFail prints a failure message and exits the program if err is not nil.
+func OnErrorFail(err error, message string) {
+	if err != nil {
+		fmt.Printf("%s: %s", message, err)
+		os.Exit(1)
```

Comment:
I recall from a previous discussion that os.Exit() will not invoke deferred procedure calls which can be troublesome (e.g. deleting a resource group in a deferred proc).  Since this is used for samples I think we can simply panic() here instead although I'm open to other ideas as well (if we do go with panic then change the function name to "OnErrorPanic").